grafana/provisioning/dashboards/dev/anchorservice.json

```diff
@@ -33,6 +33,727 @@
             "title": "Containers",
             "type": "row"
         },
+        {
+            "aliasColors": {},
+            "bars": false,
+            "dashLength": 10,
+            "dashes": false,
+            "datasource": "CloudWatch",
+            "fieldConfig": {
+                "defaults": {},
+                "overrides": []
+            },
+            "fill": 1,
+            "fillGradient": 0,
+            "gridPos": {
+                "h": 7,
+                "w": 6,
+                "x": 0,
+                "y": 1
+            },
+            "hiddenSeries": false,
+            "id": 49,
+            "legend": {
+                "avg": false,
+                "current": false,
+                "max": false,
+                "min": false,
+                "show": true,
+                "total": false,
+                "values": false
+            },
+            "lines": true,
+            "linewidth": 1,
+            "nullPointMode": "null",
+            "options": {
+                "alertThreshold": true
+            },
+            "percentage": false,
+            "pluginVersion": "7.5.7",
+            "pointradius": 2,
+            "points": false,
+            "renderer": "flot",
+            "seriesOverrides": [],
+            "spaceLength": 10,
+            "stack": false,
+            "steppedLine": false,
+            "targets": [
+                {
+                    "alias": "",
+                    "dimensions": {
+                        "ClusterName": "ceramic-dev-cas",
+                        "TaskDefinitionFamily": "ceramic-dev-cas-anchor"
+                    },
+                    "expression": "",
+                    "id": "",
+                    "matchExact": true,
+                    "metricName": "CpuUtilized",
+                    "namespace": "ECS/ContainerInsights",
+                    "period": "",
+                    "refId": "A",
+                    "region": "default",
+                    "statistics": [
```

Comment:
this does seem like the same number being counted in pr 48 at https://github.com/3box/ceramic-stats/pull/48/files#diff-c5c6048e4df66fd4bf2d0c16357ff399cbe9b268aa2e6c5bf12d90a15285e98bR196

Reply:
yes that's correct. it is the same as what you have. I'll merge this after you do yours and resolve the conflicts